test/fixtures/virtual-module/unplugin.js

```diff
@@ -4,16 +4,16 @@ module.exports = createUnplugin(() => {
   return {
     name: 'virtual-module-fixture',
     resolveId(id) {
-      return id.startsWith('virtual/') ? id : null
+      return id.startsWith('virtual/') ? `/__${id}` : null
```

Comment:
I don't think we should change the fixtures. There would be plugins serve virtual modules this way and we should keep supporting it

Reply:
I have rollback the fixture in the tests.

Reply:
Thank you for your comment,  but I’m a bit confused about whether this means should modify the fixture in the tests or let the plugin authors handle the platform differences?